spec/requests/api/go_cardless/go_cardless_spec.rb

```diff
@@ -286,7 +286,7 @@
                 metadata: {
                   customer_id: customer_id # a_string_matching(/\ACU[0-9A-Z]+\z/)
                 },
-                charge_date: "2016-06-20"
```

Comment:
These are GBP params - are they used in any GBP charge date specs? GBP charge dates need to match the 20th of the month.